### PR TITLE
enable tag (in addition to branch) in seed-jobs configuration

### PR DESCRIPTION
**Pull request by yorammi**

### CODE REVIEW

**Comment by yorammi** [comment]
@odavid it is odd that the travis build failed for something I didn't touch in the PR code. Can you check?

File: config-handlers/SeedJobsConfig.groovy [hunk 1]
```diff
@@ -51,8 +51,8 @@ def seedJobConfig(config){
             hudson.plugins.git.GitSCM.createRepoList(
                 source?.remote, 
                 source?.credentialsId
-            ), 
-            source?.branch ? [new hudson.plugins.git.BranchSpec("*/${source?.branch}")] : [], 
+            ),
+                source?.branch ? [new hudson.plugins.git.BranchSpec("*/${source?.branch}")] : (source?.tag ? [new hudson.plugins.git.BranchSpec("${source?.tag}")] : []),
```
Comment (odavid) [inline config-handlers/SeedJobsConfig.groovy]: @yorammi - let us support both tags and branches in the same way: just remove the `*/` .
```source?.branch ? [new hudson.plugins.git.BranchSpec("${source?.branch}")] : [],```

Please also fix the indentation...

Comment (yorammi) [reply config-handlers/SeedJobsConfig.groovy]: @odavid , I really wondered why we need the * there anyway... in my Jenkins servers, I usually don't put it when I configure a Jenkinsfile from a branch. So, will it be OK to just remove it without adding the 'tag' option?

Comment (odavid) [reply config-handlers/SeedJobsConfig.groovy]: Yes

Comment (yorammi) [reply config-handlers/SeedJobsConfig.groovy]: In that case, I think I'll delete this PR and create a new one from the latest version

**Pull request closed without merging**